File: ExcelSheetColumnNumber/daewook.java
public class Solution {
    public int titleToNumber(String s) {
      int result = 0;
      int strLen = s.length();

    	for (int i =0; i<strLen; i++) {
    		result += Math.pow(26, strLen - i - 1) * (s.charAt(i) - 'A' + 1);
    	}

    	return result;
    }
}

File: ExcelSheetColumnNumber/daewook.py
class Solution(object):
    def titleToNumber(self, s):
        """
        :type s: str
        :rtype: int
        """
        res = 0
        strLen = len(s)
        for idx in range(strLen):
            res += pow(26,strLen-idx-1)*(ord(s[idx])-ord('A')+1)
        return res
